templates/CRM/Campaign/Form/addCampaignToComponent.tpl

```diff
@@ -5,9 +5,9 @@
 {* add campaign in component search *}
 <tr class="{$campaignTrClass}">
     {assign var=elementName value=$campaignInfo.elementName}
-
-    <td class="{$campaignTdClass}">{$form.$elementName.label}<br />
-    <div class="crm-select-container">{$form.$elementName.html}</div>
+    <td class="{$campaignTdClass}">
+      <label>{$form.$elementName.label}</label>
+      {$form.$elementName.html}
```

Comment:
I feel good about this PR except for the changes to this file. This isn't directly related to events or dates, and i'm not sure about the change. Quickform already generates a `<label>` tag so I think this would result in `<label><label>Label</label></label>`